test/util.spec.js

```diff
@@ -2198,4 +2198,157 @@ describe('UTIL', function() {
+                            sponsoredBy: {
+                                required: true
+                            },
+                            body: {
+                                required: true
+                            }
+                        }
+                    },
+                    "DIV_2":{
+                        "nativeOnly": true,
+                        config: {
+                            image: {
+                                required: true,
+                                sizes: [150, 50]
+                            },
+                            title: {
+                                required: true,
+                                len: 80
+                            },
+                            sponsoredBy: {
+                                required: true
+                            },
+                            body: {
+                                required: true
+                            }
+                        }
+                    }
+                }
+            };
+            sizes = [[300,250]];
+            currentSlot = { 
+                getSizes: function(){
+                    return [[300,250]];
+                },
+                getAdUnitID: function(){
+                    return "testAdUnit";
+                },
+                getDivID: function() {
+                    return commonDivID;
+                },
+            }
+            sinon.spy(currentSlot, "getDivID");
+            sinon.spy(currentSlot, "getSizes");
+            sinon.spy(currentSlot, "getAdUnitID");
+            done();
+        });
+
+        afterEach(function(done) {
+            nativeConfiguration = null;
+            sizes = null;
+            commonDivID = "DIV_1"
+            done();
+        });
+
+        it('is a function', function(done) {
+            UTIL.getMediaTypeObject.should.be.a('function');
+            done();
+        });
+
+        it('should return mediaTypeObject with Native and Banner if config is present',function(done){
+            var expectedResult =  { 
+                native: {
+                    image: {
+                        required: true,
+                        sizes: [150, 50]
+                    },
+                    title: {
+                        required: true,
+                        len: 80
+                    },
+                    sponsoredBy: {
+                        required: true
+                    },
+                    body: {
+                        required: true
+                    }
+                },
+                banner: {
+                    sizes: sizes
+                }
+            }
+            var result = UTIL.getMediaTypeObject(nativeConfiguration, sizes, currentSlot)
+            result.should.deep.equal(expectedResult);
+            done();
+        });
+        
+        it('should return mediaTypeObject with Native only if for that kgpv only native is required',function(done){
```

Comment:
please mention the name of the flag in the description "nativeOnly"

Reply:
Done